backend/requirements-dev.txt

```diff
@@ -190,3 +190,6 @@ wheel==0.45.1
 # The following packages are considered to be unsafe in a requirements file:
 # pip
 # setuptools
+pyclamd

```

Comment:
Ah, ok. I see here that the clamav modules are included in requirements text file.

Just wondering why we added 'factory-boy' here?

Reply:
Looks like these were added directly into this file as well, @Joshua-Tihabwangye :) Totally fine that this is a bit confusing. Was for me when we started doing it 😊 See the directions on how to update dependencies at the top of [backend/requirements.in](https://github.com/activist-org/activist/blob/main/backend/requirements.in), and please let us know if you have any questions! This uses [pip-tools](https://github.com/jazzband/pip-tools) :)